### ISSUE TITLE
Peer Review

### ISSUE THREAD

**Issue by HuuDatDo** [GitHub-Traveler/Project-1---COMP4010#1]
- What is the scale and structure of electricity generation of the 5 countries with the highest (energy consumption/electricity generation/GDP) and the world (Possibly in the year 2023)? --> This question lists the data in a graph. Data is shown as evidence to prove or state something, so you could upgrade for example find some data on how rich the top 5 countries are, and imply the consumption related to richness.
- Same with question number 2, Can you use your data on Vietnam to show how fast/slow our country is growing? or can you use your data to show how efficiently Vietnam uses resources by comparing it to other countries?
- There is much room to investigate further, we suggest you further explore the trend through the years the correlation between different energy sources, and how the trend affects the GDP of countries.

**Comment by tienvu95** [comment]
I agree that for qn 2, showing only the number for Vietnam has little meaning, we need to compare ourselves with top countries? countries in the same region? top in Asia? etc. It's hard to conclude about the scale and structure of Vietnam if we don't know how good are other countries

**Comment by AaronCerty** [comment]
I already updated q2 to better compare Vietnam with other regions in term of energy source shifting.
